libheif/bitstream.h

```diff
@@ -411,10 +422,21 @@ class BitReader
     return ((int64_t) bytes_remaining) * 8 + nextbits_cnt;
   }
 
+  void set_start_offset(uint64_t offset)
+  {
+    start_offset = offset;
+  }
+
+  uint64_t get_file_offset() const
+  {
+    return start_offset + (data_length - bytes_remaining - (nextbits_cnt / 8));
+  }
+
```

Comment:
Could you please remove the `set_start_offset()` function from here and simply add the offset in the `mini`-parser? It seems unnatural to me to move this offset variable into the `BitReader` class.